sdk/identity/Azure.Identity/samples/ClientSideUserAuthentication.md

```diff
@@ -72,24 +73,31 @@ catch (AuthenticationRequiredException e)
 
 ## Persisting user authentication data
 
-Quite often applications desire the ability to be run multiple times without having to reauthenticate the user on each execution. This requires that data from the original authentication be persisted outside of the application memory, so that it can authenticate silently on subsequent executions. Specifically two pieces of data need to be persisted, the `TokenCache` and the `AuthenticationRecord`.
+Quite often applications desire the ability to be run multiple times without having to re-authenticate the user on each execution.
+This requires that data such a token or `AuthenticationRecord` from the original authentication be persisted outside of the application memory so that it can authenticate silently on subsequent executions.
```

Comment:
```suggestion
This requires that data from credentials be persisted outside of the application memory so that it can authenticate silently on subsequent executions.
```